src/System.Numerics.Vectors/src/System/Numerics/Vector.netcoreapp.tt

```diff
@@ -0,0 +1,101 @@
+﻿<#@ template debug="true" hostSpecific="true" #>
+<#@ output extension=".cs" #>
+<#@ Assembly Name="System.Core.dll" #>
+<#@ Assembly Name="System.Xml.dll" #>
+<#@ import namespace="System" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Runtime.InteropServices" #>
+<#@ include file="GenerationConfig.ttinclude" #><# GenerateCopyrightHeader(); #>
+
+namespace System.Numerics
+{
+    /* Note: The following patterns are used throughout the code here and are described here
```

Comment:
This comment appears to be duplicated with https://github.com/dotnet/corefx/blob/master/src/System.Numerics.Vectors/src/System/Numerics/Vector.tt, can we just reference that comment here?  That way the documentation stays in one place.

Reply:
I have created a T4 function and called it in both places.